charts/konk/templates/etcd-secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{- /* TODO: replace with kubeadm certs */ -}}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "konk.fullname" . }}-etcd-cert
+  labels:
+    {{- include "konk.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
```

Comment:
hooks aren't required. K8s will prevent the pod from initializing until this secret is created. If we do use them, please add a comment that delete-policy is mandatory when using pre-install hook